carbonmark-api/test/routes/projects/[id]/get.test.ts

```diff
@@ -45,7 +47,7 @@ describe("GET /projects/:id", () => {
       .post("", /.*getActivitiesByProjectId.*/i)
       .reply(200, {
         data: {
-          activities: fixtures.activities,
+          activities: fixtures.marketplace.activities,
```

Comment:
Why not use the same pattern as the one you used for mockCmsProject ?
ie add
 `const mockMarketplaceActivities = fixtures.marketplace.activities;`
and reuse that variable.

Reply:
Good idea, done.